apps/E2E/src/Checkbox/specs/Checkbox.spec.win.ts

```diff
@@ -27,17 +26,34 @@ describe('Checkbox Accessibility Testing', () => {
   });
 
   it('Checkbox - Validate accessibilityRole is correct', async () => {
-    await expect(await CheckboxPageObject.getAccessibilityRole()).toEqual(CHECKBOX_A11Y_ROLE);
+    await expect(
+      await CheckboxPageObject.compareAttribute(CheckboxPageObject._primaryComponent, Attribute.AccessibilityRole, CHECKBOX_A11Y_ROLE),
+    ).toBeTrue();
+
     await expect(await CheckboxPageObject.didAssertPopup()).toBeFalsy(CheckboxPageObject.ERRORMESSAGE_ASSERT);
   });
 
   it('Checkbox - Set accessibilityLabel', async () => {
-    await expect(await CheckboxPageObject.getAccessibilityLabel(ComponentSelector.Primary)).toEqual(CHECKBOX_ACCESSIBILITY_LABEL);
+    await expect(
+      await CheckboxPageObject.compareAttribute(
+        CheckboxPageObject._primaryComponent,
+        Attribute.AccessibilityLabel,
+        CHECKBOX_ACCESSIBILITY_LABEL,
+      ),
+    ).toBeTrue();
+
     await expect(await CheckboxPageObject.didAssertPopup()).toBeFalsy(CheckboxPageObject.ERRORMESSAGE_ASSERT);
   });
 
   it('Checkbox - Do not set accessibilityLabel -> Default to Checkbox label', async () => {
```

Comment:
nit "Do not set accessibilityLabel -> Validate accessibilityLabel defaults to label"